install/Knative-with-OpenShift.md

```diff
@@ -11,168 +11,154 @@ You can find [guides for other platforms here](README.md).
 
-Run the following to install Istio:
+  > **IMPORTANT:** Avoid using `default` project in OpenShift for deploying Knative applications. As OpenShift deploys few of its mission critical applications in `default` project, its safe not to touch to avoid any crash to the OpenShift.
 
-```shell
-curl -L https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml \
-  | sed 's/LoadBalancer/NodePort/' \
-  | oc apply -f -
-```
+### Installing Istio
 
-Monitor the Istio components until all of the components show a `STATUS` of
-`Running` or `Completed`:
+Knative depends on Istio. The [istio-openshift-policies.sh](scripts/istio-openshift-policies.sh) does run the required commands to configure necessary [privileges](https://istio.io/docs/setup/kubernetes/platform-setup/openshift/) to the service accounts used by Istio.
 
 ```shell
-oc get pods -n istio-system
+bash <(curl -s https://raw.githubusercontent.com/knative/docs/master/install/scripts/istio-openshift-policies.sh)
 ```
 
-It will take a few minutes for all the components to be up and running; you can
-rerun the command to see the current status.
+1. Run the following to install Istio:
 
-> Note: Instead of rerunning the command, you can add `--watch` to the above
-  command to view the component's status updates in real time. Use CTRL+C to exit watch mode.
+    ```shell
+    curl -L https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml \
+    | sed 's/LoadBalancer/NodePort/' \
+    | oc apply -f -
+    ```
+2. Monitor the Istio components until all of the components show a `STATUS` of `Running` or `Completed`:
 
-## Installing Knative Serving
+    ```shell
+    oc get pods -n istio-system -w
+    ```
 
-Next, we'll install [Knative Serving](https://github.com/knative/serving).
+> **NOTE:** It will take a few minutes for all the components to be up and running. Use CTRL+C to exit watch mode
 
-First, run the following to grant the necessary privileges to the service
-accounts istio will use:
+## Install Knative Serving
 
-```shell
-oc adm policy add-scc-to-user anyuid -z build-controller -n knative-build
-oc adm policy add-scc-to-user anyuid -z controller -n knative-serving
-oc adm policy add-scc-to-user anyuid -z autoscaler -n knative-serving
-oc adm policy add-scc-to-user anyuid -z kube-state-metrics -n monitoring
-oc adm policy add-scc-to-user anyuid -z node-exporter -n monitoring
-oc adm policy add-scc-to-user anyuid -z prometheus-system -n monitoring
-oc adm policy add-cluster-role-to-user cluster-admin -z build-controller -n knative-build
-oc adm policy add-cluster-role-to-user cluster-admin -z controller -n knative-serving
-```
+The following section details on deploying [Knative Serving](https://github.com/knative/serving) to OpenShift.
 
-Next, install Knative:
+The [knative-openshift-policies.sh](scripts/knative-openshift-policies.sh) does run the required commands to configure necessary [privileges] to the service accounts used by Knative.
```

Comment:
"does run the required commands to configure necessary" -->
"runs the required commands to configure the necessary"

Reply:
done